homeassistant/components/ecobee/__init__.py

```diff
@@ -116,6 +125,26 @@ def __init__(self, hass, entry, api_key, refresh_token):
         _LOGGER.error("Error updating ecobee tokens")
         return False
 
+    async def request(self, method, *args, **kwargs):
+        """Wrap calls to the ecobee API to catch token errors and retry."""
+        req = functools.partial(method, *args, **kwargs)
+        try:
+            return await self._hass.async_add_executor_job(req)
+        except ExpiredTokenError:
+            _LOGGER.warning(
```

Comment:
Is this a warning? Isn't refreshing the token a normal procedure after the token has expired? I propose debug level.

Reply:
Agreed, this should move to debug level.